components/resources/ai_chat_ui_strings.grdp

```diff
@@ -220,4 +220,22 @@
   <message name="IDS_CHAT_UI_LEO_SETTINGS_TOOLTIP_LABEL" desc="A label for leo settings tooltip">
     Leo settings
   </message>
+  <message name="IDS_CHAT_UI_WELCOME_GUIDE_TITLE" desc="A title for welcome guide component presented for first time users">
+    Hi, I'm Leo! <ph name="BEFORE">$1</ph>An AI-powered intelligent assistant, built right into Brave.<ph name="AFTER">$2</ph>
+  </message>
+  <message name="IDS_CHAT_UI_SITE_HELP_CARD_TITLE" desc="This is the title for a user prompt card designed to offer assistance with website-related tasks.">
+    Need help with a website?
+  </message>
+  <message name="IDS_CHAT_UI_SITE_HELP_CARD_DESC" desc="This description informs the user that the chatbot is capable of assisting with various website-related tasks such as summarizing articles and elaborating on website content. There is no specific call to action in this message.">
+    I can help you summarizing articles, expanding on a site's content and much more.
+  </message>
+  <message name="IDS_CHAT_UI_SITE_HELP_CARD_WITH_ACTION" desc="This description provides similar information to the previous message but concludes with a specific call to action or suggestion for the user to get started with assistance.">
+    I can help you summarizing articles, expanding on a site\'s content and much more. Not sure where to start? Try this:
```

Comment:
why the escape char?

Reply:
copy pasted directly from storybook locale, will remove it